### DISCUSSION TITLE
导出为makedown的时候自动在"["前面添加"\"

### DISCUSSION THREAD

**Discussion by iisnot** [windingwind/zotero-better-notes#692] [Q&A] [answered]
使用better notes提取论文模板后，想要导入ob中，但他会自动在"["前面添加"\"
![image](https://github.com/windingwind/zotero-better-notes/assets/88778563/e40288a0-735e-40e6-ba50-a85811c8b675)

我想要的格式是这样的
![image](https://github.com/windingwind/zotero-better-notes/assets/88778563/2a850cd8-4541-4097-8d59-72b3861566f7)

**Comment by iisnot** [comment]
![image](https://github.com/windingwind/zotero-better-notes/assets/88778563/1cdcb46b-ce1f-4c03-8779-ccfa62e8d775)
现在这个转换方括号在哪

**Comment by windingwind** [comment, marked as answer]
https://github.com/windingwind/zotero-better-notes/discussions/628

**Reply by iisnot** [reply]
zotero6 无法安装测试版
显示
无法安装插件“%S”。它可能无法与该版本的Zotero兼容

**Reply by windingwind** [reply]
readme#install
for z7 only

**Reply by iisnot** [reply]
z7 的模板不支持markdown格式
